src/components/control/date.jsx

```diff
@@ -155,6 +179,7 @@ const Date = forwardRef(function Date(props, forwardedRef) {
       style={{ height: '100%', position: 'relative' }}
     >
       <input
+        aria-labelledby={ids.label}
```

Comment:
AFAIK, the `<label>`'s `htmlFor` attribute points to the `id` here. 

But, does this improve table cell accessibility? We might have been missing some better testing patterns and/or QA patterns in regards to a11y.

Reply:
I trashed this